contracts/src/v0.8/ccip/test/pools/GHO/GhoTokenPoolRemote.t.sol

```diff
@@ -500,12 +500,10 @@ contract GhoTokenPoolRemote_setRateLimitAdmin is GhoTokenPoolRemoteSetup {
 }
 
 contract GhoTokenPoolRemote_directMint is GhoTokenPoolRemoteSetup {
-  function testDirectMintAdminSuccess(uint256 amount) public {
+  function testFuzzDirectMintSuccess(uint256 amount) public {
     amount = bound(amount, 1, type(uint128).max); // current pool capacity
-    // function expected to match bucket level of old facilitator which burns
-    // provided tokens to reduce its capacity
+
     address oldFacilitator = makeAddr("oldFacilitator");
```

Comment:
i'd rename this variable to something more generic but no big deal. And we can keep the bucket level assert

Reply:
[ca12f63](https://github.com/aave/ccip/pull/20/commits/ca12f6398157d7a0526bc3f153d69566884d9dd0)